components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/FxaAccountManager.kt

```diff
@@ -71,6 +71,7 @@ import kotlin.coroutines.CoroutineContext
  */
 internal object GlobalAccountManager {
     private var instance: WeakReference<FxaAccountManager>? = null
+    private var lastAuthErrorCheckPoint: Long = 0L
```

Comment:
*If* we wanted to avoid potential issues with concurrently triggering multiple auth errors, we could make this an array of timestamps for the last N auth errors, and then only allow at most N within the last WINDOW seconds. That's substantial extra complexity though, which may not be worth it.

(I can think of a more thorough approach, but it would require cooperation from the underlying Rust library, and would require callers to pass back the actual token from which they received an auth error rather than just calling a generic callback)

Reply:
I've switched to a simple sliding window, that should help with the multiple errors case.